cmd/geth/main.go

```diff
@@ -287,11 +287,11 @@ func startNode(ctx *cli.Context, stack *node.Node) {
 				status, _ := event.Wallet.Status()
 				log.Info("New wallet appeared", "url", event.Wallet.URL(), "status", status)
 
+				derivationPath := accounts.DefaultBaseDerivationPath
```

Comment:
great DRYing this out! Just a bit sad there is now this one useless assign in some cases - in Kotlin you can do :

```
derivationPath = if ( event.Wallet.URL().Scheme == "ledger" ) {
accounts.DefaultLedgerBaseDerivationPath
} else {
accounts.DefaultBaseDerivationPath
}
```

Reply:
Yeah, Golang don't support if else assign